.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       - run:
           name: build NumPy
           command: |
-            python3.11 -m venv venv
+            python -m venv venv
```

Comment:
I would prefer to leave this as is. We secifically must build the dicumentation with python 3.11 since we have distutils extensions. I wouldn’t want the version to change out from under us without it causing an obvious error